plugins/renesassimulatoradapter/renesassimulatoradapter_test.go

```diff
@@ -135,13 +135,13 @@ func TestMain(m *testing.M) {
 
 	connection, _, err = websocket.DefaultDialer.Dial("ws://localhost:9000", nil)
 	if err != nil {
-		log.Fatalf("Can't connect to simulator adapter: %s", err)
+		log.Errorf("Can't connect to simulator adapter: %s", err)
```

Comment:
No need to continue if connection is not started.

Reply:
added return